app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -845,11 +683,7 @@ watchEffect(() => {
   margin-right: 4px;
 }
 
-.draggable {
-  cursor: grab;
-}
-
 .dragged {
```

Comment:
When we want to apply draggable styles when a user drags an item, there are some caveats we need to consider:

1. Element can't keep up the mouse, eg when render takes more than 16ms
2. User drags away the draggable element

So it's better to either apply draggable to the whole document, or use something like 

```css
body:has(.dragged) {
   cursor: grabbing !important;
}
```


https://github.com/enso-org/enso/assets/61194245/53905526-1737-48ab-923c-47fefd6e7adf

Reply:
We are using `setPointerCapture`, so the cursor style doesn't depend on what node is under the pointer. I think it is enough to apply `dragged` to the draggable elements.

Reply:
TIL!